psalm.xml

```diff
@@ -14,6 +14,8 @@
     </projectFiles>
 
     <issueHandlers>
+		<MissingImmutableAnnotation errorLevel="suppress" />
```

Comment:
We should really have these - they are easy to add, and I'd prefer to have them in baseline (visible) than in here, since immutability is extremely important, where declared.

Reply:
https://github.com/Roave/BetterReflection/issues/754